conn.go

```diff
@@ -788,6 +781,31 @@ func (c *Conn) Handshake() Alert {
 		if connected {
 			c.state = state.(stateConnected)
 			c.handshakeComplete = true
+
+			if !c.isClient {
+				// Send NewSessionTicket if configured to
+				if c.config.SendSessionTickets {
```

Comment:
Combine this `if` statement with the previous.
```
if !c.isClient && c.config.SendSessionTickets {...}
```

Reply:
This isn't right, because the early data copy happens inside !c.isClient.

Reply:
If that's your intent, then you need to move the early data copy up into `!c.isClient`.  Right now, it's parallel.